crt_portal/cts_forms/forms.py

```diff
@@ -71,32 +71,53 @@ class Meta:
         ]
 
 
-class WhatHappened(ModelForm):
+def retrieve_or_create_choices():
+    choices = []
+    for choice in PROTECTED_CLASS_CHOICES:
+        try:
+            c = ProtectedClass.objects.get_or_create(protected_class=choice)
```

Comment:
@ilodidoj and I had questions about the variable names here — because `choices` and `choice` are already variable names, it wasn't immediately clear to us what `c` stands for. Something like `protected_class_objects`? A more semantic variable name here could help readers of the code.